### PR TITLE
RA- 1721 Unnecessary text "clear" displayed

### PR DESCRIPTION
**Pull request by sherrif10**
First commit was to update some dependencies to make the application works better

### CODE REVIEW

**Comment by sherrif10** [comment]
My commits seems to be the same but i tried to add those versions for stability of the module, this time if its merged , one is allowed to use it without recent modules cc @dkayiwa  @mogoodrich 
   Am squashing new commits

File: omod/pom.xml [hunk 1]
```diff
@@ -218,6 +219,7 @@
                     <execution>
                         <goals>
                             <goal>exec</goal>
+
```
Comment (ODORA0) [inline omod/pom.xml]: For consistency, you wouldn't want this white space here

**Comment by ODORA0** [comment]
Did you add the ticket ID to the issue you are working on?https://wiki.openmrs.org/display/docs/Pull+Request+Tips

File: omod/pom.xml (hunk 1 above)
Comment (sherrif10) [reply omod/pom.xml]: Sure @ODOROA am still working on this ticket and i hope to squash with new changes today thanks

**Comment by sherrif10** [comment]
Any one to review my commits and guidance is highly appreciated cc.@ruhanga,@mozzy @reagan-meant , @ODORA0

**Pull request closed without merging**